### ISSUE TITLE
Provide a unique Jupyter per chapter

### ISSUE THREAD

**Issue by joseberlines** [ines/spacy-course#25]
As an enhancement I suggest to provide as well an unique jupyter notebook with all the code per chapter. 
Some of us learn just better with the jupyter in front and playing around with the code and making our own annotations.
thanks

**Comment by mathematicalmichael** [comment]
I think there’s a more polite way to request this... 
like, requesting instead of suggesting. 

Just a thought. If you don’t appreciate the work of this incredible front-end, then perhaps this isn’t the right repo to learn spacy.

**Comment by joseberlines** [comment]
Dear Michael,
I am really sorry my message came across this way. I am a complete beginner in GitHub, perhaps I should have said that, I don't even know how to tag my comment, plus to be honest for a software beginner the difference between "requesting" instead of "suggesting" is not so clear.

That said, I am more than appreciating spacy and the work done in this library that I considered unbeliable.
I said that several times in my twitter account replying and retweeting Ines tweets.
I am joining the conference and work day in berlin in a couple of months, and there you see how much I appreciate spacy and the course you put up there in spacy.io.

Moreover, the suggestion or request (take it as you wish) I am doing myself already. I have the jupyter notebooks for the 3 first chapters, so I can (somehow) send it to you once they are properly edited. No problem whatsoever. As a beginner in NLP and python I simply have difficulties to believe I can somehow contribute. But if you think it might be a good idea propose me a way to send you the jupiters. I hope to finish the course in two weeks.

In short, take my comment in the following way: With jupyter notebooks I work a lot playing around with every snippet of the code, and run it cell by cell, having this way of learning/working might not be usual, but it is just another one, and having the code in one jupyter might help, not me, but others, since as I say, I do it myself already manually.

You can close this issue. no problem. Sorry for the inconvenience caused.

**Comment by mathematicalmichael** [comment]
Hey man, no need to apologize to me. This isn't my repo, just one that I happen to be watching because it's so good.

To clarify, what you are doing is completely natural and fine. Keep learning the way that best suits you. Didn't mean to scare you (sounds like it was just a translation mishap.. "suggest" carries a different implication than "request", since one puts you in a position of power and the other does not), just wanted put on your radar that Github is a place where politeness does go a long way.

This repository has a little less to do with "learning spacy" so much as it sets up an amazing website that allows for interactive learning and leverages jupyter on the back-end. Technically, this project can be forked and turned into a C++ course, or Julia. Notebooks don't translate nearly as well to mass-adoption as this kind of front-end interface does.

**Comment by joseberlines** [comment]
No problem,
as I said when I finish to learn the course I will paste the whole text of the jupyter there.

**Comment by ines** [comment]
See here 🙂 https://github.com/cristianasp/spacy
